### PR TITLE
Minor updates to comments for some verbiage that gets flagged by political correctness tools

### PR DESCRIPTION
**Pull request by ShiningMassXAcc**
Replaces two terms used in comments with some more politically correct alternatives.

### CODE REVIEW

**Review by LilyWangLL — APPROVED** [review]

**Comment by LilyWangLL** [comment]
Just modify the comments, so I canceled the CI pipeline run.

**Comment by dg0yt** [comment]
Not ready for merge.

> Just modify the comments, so I canceled the CI pipeline run.

Bad idea. CI would catch the missing port version bump for qtbase which the reviewer didn't notice.
And CI build time would show the price the change has for vcpkg user.

**Comment by dg0yt** [comment]
And please don't delay the other qtbase PRs over it!

**Comment by dg0yt** [comment]
The qtbase change was cherry-picked into #38682.

File: ports/qtbase/cmake/qt_install_submodule.cmake [hunk 1]
```diff
@@ -150,7 +150,7 @@ function(qt_cmake_configure)
         ${disable_parallel}
         OPTIONS
             -DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON
-            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to screw with users toolchain settings.
+            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to mess with users toolchain settings.
```
Comment (LilyWangLL) [inline ports/qtbase/cmake/qt_install_submodule.cmake:153]:
```suggestion
            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to screw with users toolchain settings.
```
This change was cherry-picked into https://github.com/microsoft/vcpkg/pull/38682. Please revert this change, if you just modify the `scripts/cmake/vcpkg_configure_make.cmake`, you don't need modify the version file.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by ShiningMassXAcc** [comment]
> Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

Thanks (I was out for a bit, so coming back here very late)!

**Comment by BillyONeal** [comment]
I cancelled this because it was going to take machine-days of time and invalidate all vcpkg customers' caches, and I don't think that's worth it to fix a comment. However, next time vcpkg_configure_make needs to be changed for any other reason I think this should be included into it.

**Comment by BillyONeal** [comment]
I merged this because I just merged another change which invalidated basically all caches. I didn't retrigger the effective world rebuild from it because it only changes a comment.
